evoting/service/service.go

```diff
@@ -174,6 +179,47 @@ func (s *Service) Login(req *evoting.Login) (*evoting.LoginReply, error) {
 	return &evoting.LoginReply{Token: token, Admin: admin, Elections: elections}, nil
 }
 
+// LookupSciper calls https://people.epfl.ch/cgi-bin/people/vCard?id=sciper
+// to convert scipers to names
+func (s *Service) LookupSciper(req *evoting.LookupSciper) (*evoting.LookupSciperReply, error) {
+	url := fmt.Sprintf("https://people.epfl.ch/cgi-bin/people/vCard?id=%s", req.Sciper)
```

Comment:
If sciper contains characters other than [0-9], then error. If len(sciper) > 20 error. We want to be polite net citizens and not pass on crazy input to them.